tests/Unit/Api/Service/MyParcelApiServiceTest.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/** @noinspection StaticClosureCanBeUsedInspection,PhpUnhandledExceptionInspection */
+
+declare(strict_types=1);
+
+use Composer\InstalledVersions;
+use MyParcelNL\Pdk\Api\Service\ApiServiceInterface;
+use MyParcelNL\Pdk\Api\Service\MyParcelApiService;
+use MyParcelNL\Pdk\Base\Factory\PdkFactory;
+use MyParcelNL\Pdk\Tests\Bootstrap\MockPdkConfig;
+use function DI\autowire;
+
+it('get correct headers', function () {
+    $pdk = PdkFactory::create(
+        MockPdkConfig::create([
+            ApiServiceInterface::class => autowire(MyParcelApiService::class)->constructor([
+                'userAgent' => 'Prestashop/1.7.8.6',
+                'apiKey'    => 'thisistheapikey',
```

Comment:
hoeft niet (wanneer je die dingen in de apiservice laat fallbacken naar null)

+ iets om op te letten, schrijf geen test strings waarvan phpstorm het niet herkent als woorden, dit wordt gemarkeerd als typo